### PR TITLE
Added v* to list of quarkus-cxf branches

### PR DESCRIPTION
**Pull request by shumonsharif**
This is to address https://github.com/quarkiverse/quarkus-cxf/issues/220

### CODE REVIEW

File: antora-playbook.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ content:
       branches: [ master ]
       start_path: docs
     - url: https://github.com/quarkiverse/quarkus-cxf.git
-      branches: [ master ]
+      branches: [master, v*]
```
Comment (gastaldi) [inline antora-playbook.yml:29]: You can also use `HEAD` instead of `master` in case the branch gets renamed

**Review by gastaldi — APPROVED** [review]

**Comment by gastaldi** [comment]
Oops I think I merged that too early

**Comment by gastaldi** [comment]
@shumonsharif I had to revert the PR. Can you reopen it?

**Comment by shumonsharif** [comment]
@gastaldi done, I submitted a new PR.

File: antora-playbook.yml (hunk 1 above)
Comment (shumonsharif) [reply antora-playbook.yml:29]: Whoops my bad, just saw this comment.  Did you want me to address it?  I think master is OK for now, though if needed I can switch it to HEAD later, if that works?

Comment (gastaldi) [reply antora-playbook.yml:29]: That's fine, I might probably send a separate PR changing all of them to `HEAD` at some point

**Pull request merged**